custom-recipes/meaningcloud-sentiment-analysis/recipe.py

```diff
@@ -1,77 +1,106 @@
 import dataiku
 import meaningcloud
 import pandas as pd
-from dataiku.customrecipe import *
-from meaningcloud_common import *
+from dataiku.customrecipe import (
+    get_input_names_for_role,
+    get_output_names_for_role,
+    get_recipe_config,
+    get_plugin_config,
+)
+from meaningcloud_common import setRequestSource, isBlockingErrorType
 
 
-#==============================================================================
+# ==============================================================================
 # PLUGIN + RECIPE SETTINGS
-#==============================================================================
+# ==============================================================================
 
-input_name  = get_input_names_for_role('input_dataset')[0]
-output_name = get_output_names_for_role('output_dataset')[0]
+input_name = get_input_names_for_role("input_dataset")[0]
+output_name = get_output_names_for_role("output_dataset")[0]
 
-input_dataset  = dataiku.Dataset(input_name)
+input_dataset = dataiku.Dataset(input_name)
 output_dataset = dataiku.Dataset(output_name)
 
 
-meaningcloud_connection = dataiku.customrecipe.get_plugin_config().get('meaningcloud_connection')
+meaningcloud_connection = get_plugin_config().get("meaningcloud_connection")
 
-license_key = meaningcloud_connection.get('license_key', None)
-server      = meaningcloud_connection.get('meaningcloud_server', 'https://api.meaningcloud.com')
-language    = get_recipe_config().get('language', None)
-text_column = get_recipe_config().get('column_name', None)
+license_key = meaningcloud_connection.get("license_key", None)
+server = meaningcloud_connection.get(
+    "meaningcloud_server", "https://api.meaningcloud.com"
+)
+language = get_recipe_config().get("language", None)
+text_column = get_recipe_config().get("column_name", None)
 
 
-#==============================================================================
+# ==============================================================================
 # AUXILIARY FUNCTIONS
-#==============================================================================
+# ==============================================================================
 
 # Analyzes the text passed as a parameter
 def analyzeText(text, lang):
     global index_count
     print("Analyzing sentiment for text #%s" % str(index_count))
 
     # this is where we are going to store our results
-    polarity = ''
+    polarity = ""
```

Comment:
If we are initializing the result variables, it'd be more coherent to initialize all of them (or remove the one we have, if not necessary).

```suggestion
    polarity = ""
    subjectivity = ""
    irony = ""
    agreement = ""
    confidence = ""
```

Reply:
Good idea, you're right